src/net.h

```diff
@@ -582,7 +554,6 @@ class CNode
     /** A ping-pong round trip has completed successfully. Update latest and minimum ping times. */
```

Comment:
```suggestion
    /** A ping-pong round trip has completed successfully. Update latest ping time. */
```

In fact, `m_last_ping_time` may as well be moved into `Peer` since it's not used in any net level logic and is only for RPC/GUI stats. Could be done as a follow-up.

Reply:
Gonna leave this for a follow-up.